### ISSUE TITLE
Suggestion: actual reasons

### ISSUE THREAD

**Issue by BogdanAlexandru** [ryanburgess/engineer-manager#22]
The contributor who adds a new resource here should also add a description of the contents of the resource and what problem(s) it helps the reader with. 

Otherwise the only way to find out if you'd be interested in a particular resource is to either evaluate its title (which is not always very helpful, I'm sure most would agree) or open the resource link (which is a complete waste of time since it implies you're doing it for ALL resources).

Basically:

* **Awesome Book**
     Reasons why you'd want to check it out.

**Comment by BogdanAlexandru** [comment]
As a side note the information is readily available for inclusion to anyone submitting a PR. For example, extracting the details of a book from its Amazon page:

> While many people talk about how great it is to start a business, very few are honest about how difficult it is to run one. Ben Horowitz analyzes the problems that confront leaders every day, sharing the insights he’s gained developing, managing, selling, buying, investing in, and supervising technology companies. 

Other people come up with the description by themselves, but leave it in the PR's comments only:

> I've tried to drop in my 2 cents by adding the list that has helped me time and again, find the correct thinking framework when stuck with complex problems involving a mix of humans and technology.
Joel Spolsky has been objective about things that matter the most in team building and man management in tech.

**Comment by ryanburgess** [comment]
Great suggestion! I've added an option to add a reason for why you're sharing a link. I'll work on adding reasons for some of the links but for now I'll close this issue.
